pkg/pipelines/bootstrap.go

```diff
@@ -311,3 +316,8 @@ func checkPipelinesFileExists(appFs afero.Fs, outputPath string, overWrite bool)
 	}
 	return nil
 }
+
+//logs Success message to the outputs
+func logBootstrapStatus(message string) {
```

Comment:
Any advantage of using this function?

Reply:
Just looks more aesthetic while logging errors!